edi_oca/models/edi_exchange_type.py

```diff
@@ -87,11 +91,18 @@ class EDIExchangeType(models.Model):
                 process:
                   usage: $comp_usage
 
+              filename_pattern:
+                force_tz: Europe/Rome
+                date_pattern: %Y-%m-%d-%H-%M-%S
+
             In any case, you can use these settings
             to provide your own configuration for whatever need you might have.
         """,
     )
     advanced_settings = Serialized(default={}, compute="_compute_advanced_settings")
+    filename_pattern_settings = Serialized(
```

Comment:
I'm not in favour of adding a serialized field for each an every key in adv settings. This will encourage others to do so and IMO there's no gain especially for something that is used in a small ctx like this.